src/app/state/rest/hooks/useRequestQueue.tsx

```diff
@@ -33,13 +33,13 @@ const reducer = produce((draft: State, action: Action) => {
   }
 })
 
-const isPending = (state: Readonly<State>, url: string): boolean =>
-  state.find(_ => _.url === url) !== undefined
+const isPending = (state: Readonly<State>, url: string, method: string): boolean =>
```

Comment:
Could we use `type Method = "GET" | "POST" | "PUT" | "PATCH" | "DELETE"`?

Reply:
Yes. I think method is already somewhere defined even.